### DISCUSSION TITLE
edit_config_file editor?

### DISCUSSION THREAD

**Discussion by jimcornmell** [kovidgoyal/kitty#7170] [General]
What editor/what is the mechanism for the configuration option `edit_config_file`.  I seem to get a new kitty instance appear for a second and then it closes.  I'm on Kubuntu.  Any suggestions?

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.editor

**Reply by jimcornmell** [reply]
Perfect works fine.  Sorry for missing this, doh!
